packages/pam/local/database-proxy.go

```diff
@@ -30,6 +34,18 @@ const (
 	ALPNInfisicalPAMCancellation ALPN = "infisical-pam-session-cancellation"
 )
 
+func askForApprovalRequestTrigger() (bool, error) {
+	prompt := promptui.Prompt{
+		Label:     "This action requires approval. You may trigger an approval request now. Continue?",
```

Comment:
thoughts on "You may create an approval request now" instead of "you may trigger.."?